### ISSUE TITLE
Not able to test with pretrained checkpoints

### ISSUE THREAD

**Issue by Shrashti** [KranthiGV/Pretrained-Show-and-Tell-model#30]
Trying running with /model.ckpt-2000000.index getting below error

NotFoundError (see above for traceback): Tensor name "InceptionV3/Conv2d_1a_3x3/BatchNorm/beta" not found in checkpoint files /Users/shrgupta5/WA-projects/Image_captioning/Experiments/ImgtoTxt_show_tell/punny_captions/Pretrained-Show-and-Tell-model-master/model.ckpt-2000000.index
         [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
